CalibTracker/SiStripHitEfficiency/src/HitEff.cc

```diff
@@ -525,20 +533,23 @@ void HitEff::analyze(const edm::Event& e, const edm::EventSetup& es){
 	  
 	  // --> Get trajectory from combinatedState 
 	  iidd = TM->monodet_id();
-	  if (DEBUG) cout << "setting iidd = " << iidd << " before checking efficiency and ";
+	  LogDebug("SiStripHitEfficiency:HitEff") << "setting iidd = " << iidd << " before checking efficiency and ";
 	  
 	  xloc = TM->localX();
 	  yloc = TM->localY();
 	  
-	  xErr =  TM->localErrorX();
```

Comment:
Warning: xErr and yErr are used in this code. They have to be defined. We just w'd like to remove them from the tree. So these 2 lines should be kept.